VMEncryption/main/ConfigFromFile.py

```diff
@@ -18,11 +18,17 @@
 
 import os
 
-def get_build_version_from_file():
-    # Note, version.txt is expected to be within the same directory as this file
-    file_path = os.path.join(os.path.dirname(os.path.realpath(__file__)), 'version.txt')
+def get_config_value_from_file(file_name):
+    # Note,the config file is expected to be within the same directory as this file
+    file_path = os.path.join(os.path.dirname(os.path.realpath(__file__)), file_name)
     if not os.path.exists(file_path):
         raise IOError('Unexpected: version.txt file cannot be found at [%s]' % file_path)
     
     with open(file_path) as text_file:
-        return text_file.read()
\ No newline at end of file
+        return text_file.read().strip()
+
+def get_build_version():
+    return get_config_value_from_file('version.txt')
```

Comment:
Will it be possible to have one file with both version and name?

Reply:
I was thinking the same but with any file format, the build code would need to understand it to be able to modify. If we keep adding config values, we should probably move them to a JSON file or so.

Reply:
Yes, consolidating any values being introduced by the build system into a single config file (rather than multiple) would be nice.

Reply:
Agree. Let me see how that would work with JSON.

Reply:
ConfigParser may also be an option - https://docs.python.org/2/library/configparser.html

Reply:
Switched to a common JSON file.